### ISSUE TITLE
unknown ArmingState 10230

### ISSUE THREAD

**Issue by austinmroczek** [craigjmidwinter/total-connect-client#173]
Initially reported at https://github.com/home-assistant/core/issues/64303

```txt
unknown ArmingState 10230 in OrderedDict([('PartitionID', 1), ('ArmingState', 10230), ('IsAlarmResponded', False), ('AlarmTriggerTimeLocalized', None), ('PartitionName', 'Main'), ('IsStayArmed', False), ('IsFireEnabled', False), ('IsCommonEnabled', False), ('IsLocked', False), ('IsNewPartition', False), ('IsNightStayEnabled', 0), ('ExitDelayTimer', 0)]);
```

**Comment by austinmroczek** [comment]
@heath-howard please tell us a bit more about your alarm system and what it's status was when you get this error.  I understand it's after you have set the system to `arm stay` (a.k.a. `arm home` in Home Assistant terms).  

Was it a standard `arm stay` or did you do some fancy custom bypass or other unique setting?

Which alarm panel hardware are you using?  

When you armed the system, were any zones faulted, troubled or bypassed?

Also, if you don't mind, please follow the troubleshooting instructions at https://github.com/craigjmidwinter/total-connect-client#troubleshooting and post the info about your system (after removing personal details).  I'm wondering if you have any other new/different codes related to your system.

**Comment by heath-howard** [comment]
Yup, only getting an error when the system is set to Arm Home. Happens irregardless of where it is set from (native Total Connect 2.0 app, Home Assistant integration or HomeKit through Home Assistant HomeKit integration, or panel itself). Arm Away works exactly as expected. 

Standard Arm Home. Nothing fancy on the panel and no zones are programmed. 

Panel is the new Honeywell PROA7Plus.  The conditions are either Arm Away or Arm Home (note not Arm Stay). 

No zones or sensors faulted or bypassed. 

I’ll try to run through the troubleshooting and generate the text file. Will post (if I can figure it out!). 

Very cool integration. Particularly like how all the sensors are exposed through the API. 

Heath Howard

**Comment by austinmroczek** [comment]
If you have time please bypass a zone and arm home.  

I'd like to see if we get the usual code for that, or if all of your arm home normal/bypass/etc all have new code numbers that throw errors.

**Comment by heath-howard** [comment]
I did bypass a sensor and armed, same result.

Pretty new to home assistant, kind of blown away how badass it is. BUT I
can't seem to figure out how to run the python troubleshooting command.  My
installation (MACOS Virtual Box install) doesn't let me exit out of the ha
> command line on the virtual machine, just kicks back to ha > prompt.

**Comment by heath-howard** [comment]
Logger: total_connect_client.partition
Source: /usr/local/lib/python3.9/site-packages/total_connect_client/partition.py:57
First occurred: 12:41:02 PM (9 occurrences)
Last logged: 12:47:59 PM

unknown ArmingState 10230 in OrderedDict([('PartitionID', 1), ('ArmingState', 10230), ('IsAlarmResponded', False), ('AlarmTriggerTimeLocalized', None), ('PartitionName', 'Main'), ('IsStayArmed', False), ('IsFireEnabled', False), ('IsCommonEnabled', False), ('IsLocked', False), ('IsNewPartition', False), ('IsNightStayEnabled', 0), ('ExitDelayTimer', 0)]): report at https://github.com/craigjmidwinter/total-connect-client/issues

**Comment by austinmroczek** [comment]
Thanks for looking into it.  Looks like we just have one new code for Armed Stay/Home.  I'll code this up as soon as I can and push a new release.  

Once that is ready we can get it fixed inside HA.

**Comment by rtm2017** [comment]
I know this is closed, but I just installed my ProA7Plus panel and ran into the same problem, googled, and found this issue. Updated to the beta branch of HA and it appears to include this fix and resolves the error! 

Thanks for the support!

**Comment by essvee0** [comment]
Apologies for commenting on a closed issue, but my issue is VERY similar, so it seems appropriate:

unknown partition ArmingState 10232 in OrderedDict([('PartitionID', 1), ('ArmingState', 10232), ('IsAlarmResponded', False), ('AlarmTriggerTimeLocalized', None)]):

I also have a new PROA7PLUS system, and the only difference from what is reported above is I am arming in Arm Stay/Home **Instant**. Thank you!

**Comment by austinmroczek** [comment]
@essvee0 thanks for reporting this.  I just started a new issue #177
